lib/stripe/util.rb

```diff
@@ -12,11 +12,15 @@ module Util
 
     # Options that should be copyable from one StripeObject to another
     # including options that may be internal.
-    OPTS_COPYABLE = (OPTS_USER_SPECIFIED + Set[:api_base]).freeze
+    OPTS_COPYABLE = (
+      OPTS_USER_SPECIFIED + Set[:api_base]
+    ).freeze
 
     # Options that should be persisted between API requests. This includes
     # client, which is an object containing an HTTP client to reuse.
-    OPTS_KEYS_TO_PERSIST = (OPTS_USER_SPECIFIED + Set[:client]).freeze
+    OPTS_PERSISTABLE = (
```

Comment:
I noticed that `OPTS_KEYS_*` didn't really follow the other naming conventions, so I renamed this to `OPTS_PERSISTABLE` so that it's a better match.